chicagoland.pandemicresponsecommons.org/portal/gitops.json

```diff
@@ -951,13 +961,13 @@
           },
           {
             "field": "hhinc_mean",
-            "name": "Mean Household Income (USD)"
+            "name": "Mean Household Income (US Currency"
```

Comment:
need a closing parenthesis...

Reply:
Czech this out: https://github.com/uc-cdis/gitops-qa/blob/master/qa-covid19.planx-pla.net/portal/gitops.json#L954

Reply:
yeah i know :p